### PR TITLE
Fix the wrong use of TestSessionService in the reporting helpers

### PR DESCRIPTION
**Pull request by jsconan**
The reporting was broken in the proctoring interface.

### CODE REVIEW

File: model/AssessmentResultsService.php [hunk 1]
```diff
@@ -127,8 +145,7 @@ public function getDeliveryData(\taoDelivery_models_classes_execution_DeliveryEx
      */
     public function getPrintableRubric(\taoDelivery_models_classes_execution_DeliveryExecution $deliveryExecution)
     {
-        /** @var TestSessionService $deliveryService */
-        $testSessionService = $this->getServiceManager()->get(TestSessionService::SERVICE_ID);
+        $testSessionService = $this->getTestSessionService();
```
Comment (hutnikau) [inline model/AssessmentResultsService.php]: Why don't you get service using `TestSessionService::singleton()` here?

Comment (jsconan) [inline model/AssessmentResultsService.php]: Yes I could do that. I took the same behavior as [`DeliveryExecutionStateService`](https://github.com/oat-sa/extension-tao-proctoring/blob/fix/reporting/model/implementation/DeliveryExecutionStateService.php#L307) to ease the access to this service for potential reuse elsewhere in the class.

I can remove it and just address the singleton if you prefer this way.

Comment (hutnikau) [inline model/AssessmentResultsService.php]: Yes i think `getTestSessionService()` is redundant there.

**Comment by jsconan** [comment]
Ok changes made.

**Pull request merged**